tests/render/test_default_markdown_view.py

```diff
@@ -133,340 +129,6 @@ def test_render_section_page():
     )
 
 
-def test_snapshot_render_section_page_with_fixture_data(validation_operator_result):
```

Comment:
No more validation operators means no more validation operator result so this test is obsolete